resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     User Navigates To Import MongoDB Provider Account On Developer View
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}          timeout=20s
     Click Element    ${errCreateDBaaSPolicy}
     Create Custom DBaaSPolicy as ${DBaaSPersona}
     User Selects Administrator View
 
+User Creates DBaaSPolicy For ${DBaaSPersona}
+    Click If Element Present    ${lblOperand}
+    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
+    Click Element    ${btnCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+Max Quota Exceeded Error Displays
+    Scroll Element Into View    ${txtQuotaError}
+    Element Should Be Visible   ${txtQuotaError}
+
+Specify Connection Namespaces And Labels
+    ${ns}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    namespace
+    ${label}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    label
+    IF      ${ns}
+        Click Element    ${btnToggleConnNS}
+        Add Policy Connection Namespaces    ${addNS}
+    END
+    IF    ${label}
+        Click Element    ${btnToggleConnNSSelect}
+        Element Should Be Visible    ${btnToggleMatchExp}
+        Click Element    ${btnToggleMatchExp}
+        Element Should Be Visible    ${btnAddMatchExp}
+        Add Policy Connection Nsselector    ${addLabel}
+    END
+
+User Creates DBaaSPolicy With ConnectionNamespace For ${namespace} As ${DBaaSPersona}
+    Set ConnectionNamespace For DBaaSPolicy
+    User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+
+Set ConnectionNamespace For DBaaSPolicy
+    Set Test Variable   ${nsAndLabel}   namespace
+    Set Test Variable   @{addNS}    ${connectionNS}
+
+DBaaSInventory Is Available For ${targetns}
+    User Filters Project ${targetns} On Project DropDown And Navigates To Database Access Page
+    Imported Provider Account Available On Database Access Screen
+    User Navigates To Add ${DBProvider} To Topology Screen
+    Select Given Provider Account On Add Topology Screen
+
+User Creates DBaaSPolicy As ${user} With ConnectionNamespace And Imports ${databaseProvider} Provider Account
+    The User Logs In To Openshift As ${user}
+    User Creates DBaaSPolicy With ConnectionNamespace For ${DBaaSTestNS} As ${user}
+    User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+User Creates DBaaSPolicy As ${user} Without ConnectionNamespace And Imports ${databaseProvider} Provider Account
+    The User Logs In To Openshift As ${user}
+    User Creates DBaaSPolicy For ${DBaaSTestNS} From Admin View As ${user}
+    User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+DBaaSInventory Is Not Available For ${targetns}
+    User Navigates To Add ${DBProvider} To Topology Screen
+    Wait Until Page Contains Element    ${txtNoDBInstance}    timeout=15s
+
+User Creates ${namespace} And Aligns ${access} Role To The ${user}
+    Set Test Variable    ${access}
```

Comment:
What does ${access} parameter means here? Edit View/Edit permissions?

Reply:
Yes, this is for the RoleBinding for the user and yes only cluster admin can do that which is part of line 174

Reply:
ok got it